modules/misunderstood/src/views/lite/utils.tsx

```diff
@@ -0,0 +1,8 @@
+import snarkdown from 'snarkdown'
+
+export const renderUnsafeHTML = (message: string = ''): string => {
+  message = message.replace(/</g, '&lt;').replace(/>/g, '&gt;')
+
+  const html = snarkdown(message)
+  return html.replace(/<a href/gi, '<a target="_blank" href')
```

Comment:
I don't think this line will ever match - there will never be a `<` because you replace them in line 4